packages/docusaurus-mdx-loader/package.json

```diff
@@ -8,8 +8,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@babel/parser": "^7.6.4",
-    "@babel/traverse": "^7.6.3",
+    "@babel/parser": "^7.7.2",
```

Comment:
We have 100% coverage test suites for rightToc at mdx-loader that uses babel parser and traverse. Seems to be passing